mods/d2k/rules/arrakis.yaml

```diff
@@ -9,8 +9,7 @@ spicebloom.spawnpoint:
 		GrowthTerrainTypes: SpiceSand
 		SpawnActor: spicebloom
 		GrowthSequences: grow0
-		GrowthDelay: 250, 750
-		RespawnDelay: 1, 2
+		Lifetime: 1750, 3250
```

Comment:
Please ignore my previous comment.
Now that we have only one delay, we wouldn't gain anything by changing the default, because the real spicebloom uses the current internal default, while this is for the spawnpoint actor.